proto/tendermint/liquidity/v1beta1/tx.proto

```diff
@@ -0,0 +1,207 @@
+syntax = "proto3";
+package tendermint.liquidity.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos_proto/coin.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+option go_package = "github.com/tendermint/liquidity/x/liquidity/types";
+
+// Msg defines the liquidity Msg service.
+service Msg {
+
+  // Submit create liquidity pool message.
+  rpc CreatePool(MsgCreatePool) returns (MsgCreatePoolResponse);
+
+  // Submit deposit to the liquidity pool batch.
+  rpc DepositWithinBatch(MsgDepositWithinBatch) returns (MsgDepositWithinBatchResponse);
+
+  // Submit withdraw from to the liquidity pool batch.
+  rpc WithdrawWithinBatch(MsgWithdrawWithinBatch) returns (MsgWithdrawWithinBatchResponse);
+
+  // Submit swap to the liquidity pool batch.
+  rpc Swap(MsgSwapWithinBatch) returns (MsgSwapWithinBatchResponse);
+}
+
+// MsgCreatePool defines an sdk.Msg type that supports submitting create liquidity pool
+//
+// See: https://github.com/tendermint/liquidity/blob/develop/x/liquidity/spec/04_messages.md
+message MsgCreatePool {
+  option (gogoproto.equal)           = false;
+  option (gogoproto.goproto_getters) = false;
+
+  string pool_creator_address = 1 [(gogoproto.moretags) = "yaml:\"pool_creator_address\"",
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      description: "account address of the origin of this message",
+      example: "\"cosmos1e35y69rhrt7y4yce5l5u73sjnxu0l33wvznyun\"",
+      format: "sdk.AccAddress"
+    }];
+
+  // id of target pool type, only 1 is allowed on this version, Must match the value in the pool.
+  uint32 pool_type_id = 2 [(gogoproto.moretags) = "yaml:\"pool_type_id\"",
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      example: "\"1\"",
+      format: "uint32"
+  }];
+
+  // reserve coin pair of the pool to deposit
```

Comment:
```suggestion
  // reserve coin pair to deposit to the pool
```

Reply:
I might be guessing wrong here, is this the reserved coin pair that will be deposited? or ?